documentatie/gebruikersdocumentatie/installeren-en-starten.md

```diff
@@ -0,0 +1,86 @@
+# Abacus installeren en starten
+
+Het installeren en starten van Abacus kan op verschillende manieren, met elk verschillende doeleinden.
+Hieronder vind je instructies voor de verschillende methodes.
+
+## Linux en macOS
+
+De onderstaande methodes werken waarschijnlijk op alle gangbare (recente) Linux-distributies en macOS, en zijn getest op Ubuntu (22.04 en hoger) en Debian (12/bookworm). 
+
+## Windows
+
+Methode 3 en 4 werken ook in Windows. Op dit moment kun je nog geen installatiebestand voor Windows downloaden, maar we zijn hiermee bezig. Zodra deze optie er is vind je hier meer info.
+
+### Methode 1: build artifact downloaden en starten
+
+Deze methode lijkt het meest op het draaien van een productiebuild en is ook het meest eenvoudig. Voer de volgende stappen uit:
+
+1. Ga in de Abacus-repository naar [Actions](https://github.com/kiesraad/abacus/actions)
+2. Klik aan de linkerkant op [Build, lint & test](https://github.com/kiesraad/abacus/actions/workflows/build-lint-test.yml)
+3. Klik aan de rechterkant op *Branch* om op een branch te filteren
+4. Selecteer de meest recente action
+5. Download het bestand *backend-build*. Let op: de download is alleen zichtbaar als je bent ingelogd in GitHub.
+
+![instructions1](/documentatie/gebruikersdocumentatie/img/build-artifact-1.png)
+![instructions2](/documentatie/gebruikersdocumentatie/img/build-artifact-2.png)
+
+De download bevat een ZIP-bestand met de binary. Deze binary bevat ook alle frontend-assets. Pak deze binary uit en start hem als volgt:
+
+```sh
+./path/to/binary/api --reset-database --seed-data
+```
+
+De argumenten zorgen ervoor dat de database wordt gereset en wordt geladen met fixtures. De kortere versie van deze opdracht is:
+
+```sh
+./path/to/binary/api -rs
+```
+
+Zie ook de help:
+
+```sh
+./path/to/binary/api --help
+```
+Het is natuurlijk ook mogelijk om Abacus zonder argumenten te starten, maar als je dat doet bevat de app geen data en zie je op de pagina het bericht *'Verkiezingen niet gevonden'*.
+
+Wanneer de API draait, klik je op de link van de API-server (<http://0.0.0.0:8080>) om naar de browser te gaan en Abacus te gebruiken.
+
+### Methode 2: script `pull-and-run` uitvoeren
+
+Dit is een Bash-script dat is bedoeld om snel een productiebuild te bouwen en starten. Het staat in de hoofdmap van de repository. Dit script maakt geen gebruik van Docker, waardoor het wel nodig is om `npm` en `cargo` geïnstalleerd te hebben.
+Instructies voor de installatie van `npm` vind je op [npm Docs](https://docs.npmjs.com/downloading-and-installing-node-js-and-npm) en instructies voor de installatie van Rust en `cargo` vind je op de [website van Rust](https://doc.rust-lang.org/cargo/getting-started/installation.html).
+
+Je kunt een git-branch als argument meegeven, zodat je snel de productiebuild van een bepaalde branch kunt starten. Als je het argument weglaat, wordt de huidige branch gebouwd.
+
+Zo start je de huidige branch:
```

Comment:
Ik denk dat het goed is om hier uit te leggen dat je de repository moet clonen ~~of downloaden~~. Je zou kunnen denken dat je alleen het script kunt downloaden en draaien.

Reply:
Blijkt dat download niet goed genoeg is, het moet een git repository zijn.
```
fatal: not a git repository (or any of the parent directories): .git
```

Reply:
Zoals besproken is downloaden ook prima wanneer Mark de wijzigingen daarvoor heeft doorgevoerd. Ik heb dit alvast beschreven in de instructies.